app/_includes/advanced.html

```diff
@@ -11,7 +11,7 @@ <h1 class="title feature-header">{{site.data[locale].advanced.title}}</h1>
   
   <div class="row contribute-options">
     <div class="columns" style="text-align: center">
-      <a href="{{ site.baseurl }}/assets/downloads/JOSM_Advanced_Mapping_EN.pdf">
+      <a href="{{ site.baseurl }}/assets/downloads/{{site.data[locale].checker.links.advancedJOSM_pdf:}}">
```

Comment:
i didn't catch this before merging but the `:` at the end inside the `}}` might cause a problem